ts/src/profilers/profile-serializer.ts

```diff
@@ -191,6 +191,18 @@ function createTimeValueType(table: StringTable): perftools.profiles.ValueType {
 }
 
 /**
+ * Returns value type for object counts (type:objects, units:count), and
+ * adds strings used in this value type to the table.
+ */
+function createAllocationCountValueType(table: StringTable):
```

Comment:
I would name it createObjectCountValueType since that's what it is according to the doc and the source code.